### PR TITLE
feat: update to denali-2.3.2

### PR DESCRIPTION
**Pull request by jkusa**
Update to denali-2.3.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@glimmer/tracking": "^1.0.3",
     "broccoli-funnel": "^3.0.1",
     "broccoli-merge-trees": "^4.1.0",
-    "denali-css": "^2.3.0",
+    "denali-css": "^2.3.2",
```
Comment (kevinhinterlong) [inline package.json:37]: `~`?

**Review by kevinhinterlong — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jkusa) [reply package.json:37]: I am hoping the 2.3.x issues was just a one off. Typically, floating the minor has works for us.

File: config/ember-try.js [hunk 2]
```diff
@@ -29,22 +29,22 @@ module.exports = async function () {
...
-      },
-      {
-        name: 'ember-canary',
-        npm: {
-          devDependencies: {
-            'ember-source': await getChannelURL('canary'),
-          },
-        },
-      },
+      // {
```
Comment (kevinhinterlong) [inline config/ember-try.js:32]: create an issue for re-enabling

**Review by kevinhinterlong — APPROVED** [review]

File: config/ember-try.js (hunk 2 above)
Comment (jkusa) [reply config/ember-try.js:32]: https://github.com/denali-design/denali-ember/issues/85

**Pull request merged**